TGC/decisions/Germany.txt

```diff
@@ -4109,7 +3946,6 @@ political_decisions = {
 				AUS = { exists = yes in_sphere = THIS }
 				KUK = { exists = yes in_sphere = THIS }
 			}
-			has_country_flag = brothers_war_winner
```

Comment:
Is this related to the GHG?

Reply:
No this is part of what obstination brought to my attention related to the inclusion of austria into germany post three hurrahs.